File: Bindings~/java/src/main/java/ro/altom/altunitytester/Notifications/AltUnityLoadSceneNotificationResultParams.java
package ro.altom.altunitytester.Notifications;

public class AltUnityLoadSceneNotificationResultParams {
    public String sceneName;
    public LoadSceneMode loadSceneMode;

    public AltUnityLoadSceneNotificationResultParams(String sceneName, LoadSceneMode loadSceneMode) {
        this.sceneName = sceneName;
        this.loadSceneMode = loadSceneMode;
    }
}

File: Assets/AltUnityTester/AltUnityDriver/Notifications/AltUnityLoadSceneNotificationResultParams.cs

namespace Altom.AltUnityDriver.Notifications
{
    public class AltUnityLoadSceneNotificationResultParams
    {
        public string sceneName;
        public AltUnityLoadSceneMode loadSceneMode;

        public AltUnityLoadSceneNotificationResultParams(string sceneName, AltUnityLoadSceneMode loadSceneMode)
        {
            this.sceneName = sceneName;
            this.loadSceneMode = loadSceneMode;
        }
    }
}